invenio_rdm_records/fixtures/tasks.py

```diff
@@ -25,8 +39,234 @@ def create_vocabulary_record(service_str, data):
 
 
 @shared_task
-def create_demo_record(data):
+def create_demo_record(data, user_id):
     """Create a demo record."""
-    service = current_rdm_records.records_service
-    draft = service.create(data=data, identity=system_identity)
-    service.publish(id_=draft.id, identity=system_identity)
+    service = current_rdm_records_service
+    identity = _identity(user_id)
+    identity.provides.add(UserNeed(user_id))
+    draft = service.create(data=data, identity=identity)
+    service.publish(id_=draft.id, identity=identity)
+
+
+@shared_task
+def create_demo_community_submission_request(
+        user_id,
+        community_data,
+        drafts_data
+):
+    """Create a community_submission requests for the param drafts."""
+    """Drafts are created in a new community."""
+    # Create a demo community
+    community = current_communities.service.create(
+        _identity(user_id),
+        community_data
+    )
+    Community.index.refresh()
+
+    # Create drafts in the community
+    for draft_data in drafts_data:
+        _create_draft.delay(
+            user_id,
+            community.data['uuid'],
+            draft_data
+        )
+
+
+@shared_task
+def _create_draft(
+        user_id,
+        community_uuid,
+        draft_data
+):
+    """Create a community_submission request."""
+    """The request is created for a draft."""
+    # Create a demo draft
+    identity = _identity(user_id)
+    identity.provides.add(CommunityNeed(community_uuid))
```

Comment:
I am not sure which use case this covers, I would guess the user does not need a community membership to create a draft, correct me if I am wrong, but that's why we have the requests, so that anyone can create a draft and then request to add it to the community?